symengine/polys/basic_conversions.h

```diff
@@ -10,14 +10,14 @@ namespace SymEngine
 // using `gen` as the genarator. Throws, if poly constructions not possible.
 // `ex` is the optional parameter for expanding the given `basic` or not.
 template <typename P>
-RCP<const P> from_basic(const RCP<const Basic> &basic,
-                        const RCP<const Basic> &gen, bool ex = false);
+RCP<const P> upoly_from_basic(const RCP<const Basic> &basic,
```

Comment:
No, I tried. Internally, they have different operations but their signatures match. Maybe I can do it using metaprogramming.